docs/user/openstack/README.md

```diff
@@ -25,13 +25,19 @@ In addition, it covers the installation with the default CNI (OpenShiftSDN), as
     - [Initial Setup](#initial-setup)
     - [API Access](#api-access)
       - [Using Floating IPs](#using-floating-ips)
+        - [Create API and Ingress Floating IP Addresses](#create-api-and-ingress-floating-ip-addresses)
+        - [Create API and Ingress DNS Records](#create-api-and-ingress-dns-records)
+        - [External API Access](#external-api-access)
+        - [External Ingress (apps) Access](#external-ingress-apps-access)
```

Comment:
It looks like your IDE added some entries to the Table of contents. Personally, I dont mind this, but I have been asked to remove these in past reviews.

Reply:
yeah I submitted another PR: https://github.com/openshift/installer/pull/4328